foundation/shared-services/core/managed-identity.tf

```diff
@@ -32,7 +32,7 @@ resource "azurerm_role_assignment" "subscription_connectivity_dns_contributor" {
 }
 
 resource "azurerm_user_assigned_identity" "external_dns_pod_identity" {
-  name                = "${var.prefix}-external-dns"
+  name                = "${var.prefix}-external-dns-${var.environment}-${var.location}"
```

Comment:
I thought about that, I believe it's a good idea personally, I just didn't go through with it. @djtjwillia What do you think?

Reply:
I think it would make sense to include it. Did we just include it because it wasn't listed? I'm not married to any particular idea but wanted to weigh in here.